docs/howto/error-reporting.txt

```diff
@@ -255,13 +255,6 @@ following methods:
     Returns ``True`` to activate the filtering operated in the other methods.
     By default the filter is active if :setting:`DEBUG` is ``False``.
 
-.. method:: SafeExceptionReporterFilter.get_request_repr(request)
```

Comment:
This is really intriguing problem. First, I find the implemetation of the example very problematic, because the `get_request_repr` modifies the request object. As for the problem they are trying to address: currently there isn't any good way to solve it. The content of the report is gathered by `ExceptionReporter.get_traceback_data` and there isn't any `DEFAULT_EXCEPTION_REPORTER` setting to specify custom `ExceptionReporter` class. I see following solutions for the problem:
 * Don't change code and just tell developers to hijack one of the methods of `SafeExceptionReporterFilter` e.g. `cleanse_special_types` to do what they need.
 * Introduce `SafeExceptionReporterFilter.get_meta_parameters` and add its result into the context for the error report template. But this solution only solves the META parameters and not other parts of the `request` (and possibly other) data in the report.
 * Introduce `DEFAULT_EXCEPTION_REPORTER` settings to allow custom `ExceptionReporter` class. To avoid new setting, it can actually replace original `DEFAULT_EXCEPTION_REPORTER_FILTER` setting either by merging `ExceptionReporterFilter` class into `ExceptionReporter` or defining filter as a class attribute of the `ExceptionReporter`. I don't quite see a need to have the implementation split into two classes in this case.  Developers could further profit from this change by getting control over the context and templates of the error reports. 

This last solution would have great impact, but IMHO would be the correct one. But we can just do the first option and leave the reporter refactoring for yet another ticket.